packages/web/src/lib/myalgowallet-types.ts

```diff
@@ -0,0 +1,214 @@
+// https://github.com/randlabs/myalgo-connect/blob/master/index.d.ts
+/* eslint-disable no-unused-vars */
+export type Address = string;
+export type Base64 = string;
+export type TxHash = string;
+
```

Comment:
Please only copy the minimum. I think only `MyAlgoConnect` and `Account has to be copied.
Other types are already in AlgoSDK, eg: https://algorand.github.io/js-algorand-sdk/classes/Transaction.html can be used instead of `Txn`.

Also, check other types defined in:
+ https://github.com/scale-it/algo-builder/blob/master/packages/web/src/types.ts

Another approach would be to use:
```
import type {...} from "@randlabs/myalgo-connect"
````
Let's see if this work - maybe we don't need this file at all :) `import type` is done only for compilation, and after compilation it is removed. so when running it should not create the weird error about undefined `window`.


Let's limit the copies to the minimum.